<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDLLDialog</class>
 <widget class="QDialog" name="SelectDLLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select DLL</string>
  </property>
  <widget class="QCheckBox" name="checkBoxCameraDLL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CameraDLL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPIODLL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PIO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLightDLL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Light DLL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSequenceDLL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequence DLL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
